backend/tests/helpers/factories/models/user_access_token.py

```diff
@@ -48,12 +49,14 @@ def _create(
             )
             db.session.add(user_orcid_id)
 
-            remote_account = RemoteAccount(
+            remote_token = RemoteToken.create(
                 user_id=user.get_id(),
                 client_id="orcid",
+                token=token,
+                secret=None,
                 extra_data={"orcid": orcid, "allow_push": allow_push},
             )
-            db.session.add(remote_account)
+            db.session.add(remote_token)
```

Comment:
Does it already https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/models.py#L240